conda.recipe/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set data = load_setup_py_data() %}
```

Comment:
how does this work?

Reply:
This is called conda-build.jinja_context, here is [code source](https://github.com/conda/conda-build/blob/master/conda_build/jinja_context.py#L103) for more details. And [here](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html?highlight=jinja#templating-with-jinja) there is some info.